plugins/invisible-typing/src/Icon.tsx

```diff
@@ -17,43 +17,41 @@ export const Icon = (props: { type?: { analyticsName: string } }) => {
   const channelWise = cfg.get("button", true) ? cfg.get("channelWise", true) : false;
   const channelsList = cfg.get("channels", { [channelId]: globalInvisible });
   const [enabled, setEnabled] = React.useState(
-    channelWise ? channelsList[channelId] : globalInvisible,
+    channelWise ? channelsList[channelId] ?? globalInvisible : globalInvisible,
   );
   return (
-    <>
```

Comment:
<a href="https://sourcery.ai"><img src="https://sourcery.ai/favicon.svg" height="15" alt="Sourcery logo" /></a>  **suggestion (code_refinement):** Removal of fragment in favor of a div with a dynamic key.

This change simplifies the DOM structure and uses the 'enabled' state to potentially trigger re-renders when the visibility state changes, which could be beneficial for keeping the UI in sync.

```suggestion
<div key={enabled ? 'enabled' : 'disabled'}>
```